machine/lenovo/lenovo_ne2572/i2ctools/001-i2ctools-sys-eeprom-16bits-address.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/sys_eeprom/onie_tlvinfo.c b/sys_eeprom/onie_tlvinfo.c
```

Comment:
For a couple of years now, the ONIE project does not accept the "i2ctools" implementation of onie-syseeprom for new platforms.  We have standardized on the busybox implementation.  Discussed here:

http://lists.opencompute.org/pipermail/opencompute-onie/2015-March/000512.html

Looking around I see I mistakenly accepted this for your `lenovo_ne10032` platform in July of this year.  That was a mistake on my part.  Missed it.

The busybox implementation is much better, more configurable without the need for platform specific patches to change things like the delay timing.  See this example:
https://github.com/opencomputeproject/onie/blob/master/machine/accton/accton_as5812_54x/busybox/conf/config

Also see these commits that added the busybox implementation:
fc76004dca594a5f4f12a7905474f77c67e5effb
481b975ac4c5e8dbec255a2ffb1af3e383475d3a

Reply:
Yea.. it was my mistake.. let me completely remove i2ctool directory